core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidator.java

```diff
@@ -157,13 +163,22 @@ private void validateVolumeType(VolumeTemplate value, Platform platform, Validat
         }
     }
 
+    private boolean isIDBrokerInstanceGroup(InstanceGroup instanceGroup) {
+        return groupNameIDBroker.equalsIgnoreCase(instanceGroup.getGroupName());
```

Comment:
nit: why use a constant and then a function, and then another helper function. Why can't we just use the string to compare directly?

Also, static strings should be upper case. 

```
IDBROKER_GROUP_NAME.equalsIgnoreCase(instanceGroup.getGroupName());
```